pkgs/io_file/src/constants.g.h

```diff
@@ -0,0 +1,95 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+// Regenerate with `dart run tool/build_constants.dart`.
+
+#include <stdint.h>
+
+// A sentinal indicating that a constant is not defined on the current platform.
+#define libc_shim_UNDEFINED 9223372036854775807
```

Comment:
Maybe add a comment that we use this value in Dart as well (and that's why it's in the .h file instead of the .c file)

Reply:
Done.